planning/motion_velocity_planner/autoware_motion_velocity_out_of_lane_module/src/types.hpp

```diff
@@ -94,19 +94,24 @@ using OutLaneletRtree = bgi::rtree<LaneletNode, bgi::rstar<16>>;
 /// @brief data related to the ego vehicle
 struct EgoData
 {
-  std::vector<autoware_planning_msgs::msg::TrajectoryPoint> trajectory_points;
+  std::vector<autoware_planning_msgs::msg::TrajectoryPoint>
+    trajectory_points;  // filtered trajectory starting from the 1st point ahead of ego
```

Comment:
Considering this [line](https://github.com/autowarefoundation/autoware.universe/pull/9209/files#diff-7bd28123bdeeef46ff15191d21a1b2d98e41109269e99fad0158976bb9a372c5R156)
It seems the filtered trajectory starts from the first point of the nearest segment, which is most likely behind ego ... (unless original trajectory starts ahead of ego)